backend/schema/validate.go

```diff
@@ -60,7 +66,7 @@ func Validate(schema *Schema) (*Schema, error) {
 
 	scopes := NewScopes()
 
-	// Validate dependencies
+	// ValidateWholeSchema dependencies
```

Comment:
Should the change to this comment and the one on line 84 be undone? It looks like "Validate" was used as a regular English language verb in these two places, rather than being a reference to the `Validate` function

Reply:
oop yes accidentally picked up from refactoring in my IDE! thanks for catching :)